cumulusci/utils/yaml/cumulusci_yml.py

```diff
@@ -0,0 +1,34 @@
+from typing import IO, Text
+from re import compile, MULTILINE
+from logging import getLogger
+from io import StringIO
+
+import yaml
+
+NBSP = "\u00A0"
+
+pattern = compile(r"^\s*[\u00A0]+\s*", MULTILINE)
+
+logger = getLogger("CumulusCI.yml")
+
+
+def _replacer_func(matchobj):
+    string = matchobj.group(0)
+    rc = string.replace(NBSP, " ")
+    if rc != string:  # this is fast for identical strings
+        logger.warn(
+            "Note: Non-breaking space character was detected in Cumulusci.yml.\n"
```

Comment:
We merge together the project config from cumulusci.yml files in several different sources. It would be ideal to identify the path to the file (which would require passing it in to cci_safe_load)

Reply:
That's another feature I removed in the interest of simplicity for this small short-term patch. I can put it back in if you feel strongly.

Reply:
Most of it will be replaced when I merge the YAML validator thing, which does take the filename and a logger argument etc..